Geocoder Example/AppDelegate.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
```

Comment:
This is not backward compatible with Xcode 9 or more accurately, iOS < 12.

Reply:
Ahh I see, would it be best to just use the old function signature then?